concepts/05 UI Components/TextArea/00 Overview.md

```diff
@@ -228,6 +228,6 @@ If an end user should _not_ be able to edit the text in the TextArea, assign **t
 - [TextArea - Handle the Keyboard Events](/concepts/05%20UI%20Components/TextArea/13%20Handle%20the%20Keyboard%20Events.md '/Documentation/Guide/UI_Components/TextArea/Handle_the_Keyboard_Events/')
 - [TextArea - Limit the Text Length](/concepts/05%20UI%20Components/TextArea/15%20Limit%20the%20Text%20Length.md '/Documentation/Guide/UI_Components/TextArea/Limit_the_Text_Length/')
 - [TextArea API Reference](/api-reference/10%20UI%20Components/dxTextArea '/Documentation/ApiReference/UI_Components/dxTextArea/')
-- [TextBox](/concepts/05%20UI%20Components/TextBox/00%20Overview.md '/Documentation/Guide/UI_Components/TextBox/Overview/')
+- [TextBox](/Documentation/ApiReference/UI_Components/dxTextBox/)
 
```

Comment:
Can you update the following lines?

Line 1:
`The TextArea is a UI component that enables a user to enter and edit a multi-line text.`

Please change "edit a multi-line text" to "edit multi-line text".

Line 78:
`By default, the TextArea checks the entered text for spelling errors.`

Please rewrite without using "By default".

`To disable this feature, assign **false** to the **spellcheck** property.`

Can we say the following instead?

`To disable this feature, set the **spellcheck** property to **false**.`

Reply:
Noted